src/build/app/settings.rs

```diff
@@ -655,6 +658,39 @@ pub enum AppSettings {
     /// This can be useful if there are many values, or they are explained elsewhere.
     HidePossibleValuesInHelp,
 
+    /// Tells `clap` to panic if help strings are omitted
+    ///  
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use clap::{App, Arg, AppSettings};
+    /// App::new("myprog")
+    ///     .setting(AppSettings::HelpRequired)
+    ///     .arg(
+    ///         Arg::with_name("foo").help("It does foo stuff")
+    ///         // As required via AppSettings::HelpRequired, a help message was supplied
+    ///      )
+    /// #    .get_matches();
+    /// ```
+    ///
+    /// # Panics
+    ///
+    /// ```rust,should_panic
+    /// # use clap::{App, Arg, AppSettings};
+    /// App::new("myapp")
+    ///     .setting(AppSettings::HelpRequired)
+    ///     .arg(
+    ///         Arg::with_name("foo")
+    ///         // Someone forgot to put .help("...") here
+    ///         // Since the setting AppSettings::HelpRequired is activated, this will lead to
+    ///         // a panic (if you are in debug mode)
+    ///     )
+    /// #   .get_matches();
+    ///```
+    ///
+    /// [``]: ./struct..html
```

Comment:
You don't need this.

Reply:
I do not need the html thing, but I do need the .get_matches() call. It is to trigger the `app._build()`. We talked about it earlier.